src/codec/MapPutCodec.ts

```diff
@@ -0,0 +1,57 @@
+import {ClientMessage} from "../ClientMessage";
```

Comment:
ClientMessage is imported like `import ClientMessage = require('..ClientMessage');`